src/components/dialogs/voltage-level-creation-dialog.js

```diff
@@ -87,228 +218,67 @@ const VoltageLevelCreationDialog = ({
-                tmpErrors,
-                makeBlock('bbs_vpos.' + bbs.idx, bbs.vertPos)
-            );
-        }
-
-        let byHorizVert = buildByHPosVPos();
-        for (let p of byHorizVert) {
-            let indices = p[1];
-            if (indices.length > 1) {
-                let msgId = 'SameHorizAndVertPos';
-                for (let idx of indices) {
-                    tmpErrors.set('bbs_hpos.' + idx, makeErrorRecord(msgId));
-                    tmpErrors.set('bbs_vpos.' + idx, makeErrorRecord(msgId));
-                }
-            }
-        }
-
-        for (let cnx of connections) {
-            addCheckToBlock(
-                tmpErrors,
-                makeBlock('cnx_fromBBS.' + cnx.idx, cnx.fromBBS)
-            );
-            addCheckToBlock(
-                tmpErrors,
-                makeBlock('cnx_toBBS.' + cnx.idx, cnx.toBBS)
-            );
-
-            if (
-                cnx.switchKind === 'DISCONNECTOR' &&
-                cnx.fromBBS === cnx.toBBS
-            ) {
-                let msgId = 'DisconnectorBetweenSameBusbar';
-                tmpErrors.set('cnx_fromBBS.' + cnx.idx, makeErrorRecord(msgId));
-                tmpErrors.set('cnx_toBBS.' + cnx.idx, makeErrorRecord(msgId));
-            }
-        }
-
-        return tmpErrors;
-    };
-
-    const updateErrors = () => {
-        let tmpErrors = collectErrors();
-
-        setErrors(tmpErrors);
-
-        return tmpErrors;
-    };
+    const inputForm = useInputForm();
+
+    const [voltageLevelId, voltageLevelIdField] = useTextValue({
+        label: 'ID',
+        validation: { isFieldRequired: true },
+        inputForm: inputForm,
+        formProps: filledTextField,
+    });
+
+    const [voltageLevelName, voltageLevelNameField] = useTextValue({
+        label: 'Name',
+        inputForm: inputForm,
+        formProps: filledTextField,
+    });
+
+    const [nominalVoltage, nominalVoltageField] = useDoubleValue({
+        label: 'NominalVoltage',
+        validation: { isFieldRequired: true },
+        inputForm: inputForm,
+        formProps: filledTextField,
+    });
+
+    const [substation, substationField] = useAutocompleteField({
+        label: 'Substation',
+        validation: { isFieldRequired: true },
+        inputForm: inputForm,
+        formProps: filledTextField,
+        values: substationOptions,
+        allowNewValue: true,
+        getLabel: getId,
+    });
+
+    const [busBarSections, busBarSectionsField] = useExpandableValues({
+        id: 'bbs',
+        labelAddValue: 'CreateBusBarSection',
+        validateItem: validateBusBarSection,
+        inputForm: inputForm,
+        formProps: filledTextField,
+        Field: BusBarSection,
+    });
+
+    const [connections, connectionsField] = useExpandableValues({
+        id: 'connections',
+        labelAddValue: 'CreateLink',
+        validateItem: validateConnection,
+        inputForm: inputForm,
+        formProps: filledTextField,
```

Comment:
Remove or add it in usseExpandableValues

Reply:
done